apps/todoapp/src/app/todos/modules/main.module.ts

```diff
@@ -24,6 +25,7 @@ const routes: Routes = [
     MainComponent,
     TodoComponent,
     FooterComponent,
+    LogoutComponent,
```

Comment:
we should avoid renaming/restructuring/refactoring while implementing new features in parallel, it took me some time till I realize that this is missing here after it was renamed from todos -> main :)